src/ui/components/ConfirmButton/index.js

```diff
@@ -67,10 +69,13 @@ export class ConfirmButtonBase extends React.Component<InternalProps> {
       confirmButtonText,
       confirmButtonType,
       i18n,
+      id,
       message,
       uiState,
     } = this.props;
 
+    invariant(id, 'The id property is required');
```

Comment:
In response to your question about adding `invariant()` for `message` and `onConfirm` (the other required props), yes, I think it would be nice to fix that while you're in here.

Reply:
yep, just wanted to be sure, thanks : )